src/compiler/crystal/config.cr

```diff
@@ -8,12 +8,16 @@ module Crystal
       version_and_sha.first
     end
 
+    def self.llvm_version
+      LibLLVM::VERSION
+    end
+
     def self.description
       version, sha = version_and_sha
       if sha
-        "Crystal #{version} [#{sha}] (#{date})"
+        "Crystal #{version} [#{sha}] (#{date}) LLVM #{llvm_version}"
       else
-        "Crystal #{version} (#{date})"
+        "Crystal #{version} (#{date}) LLVM #{llvm_version}"
```

Comment:
Maybe put it on a new line?

```
Crystal 0.21.0 (2017-02-20)
LLVM 4.0
```

My thinking is that it would make it easier to report other info in the future (stick each item on a new line).